text/0007-weekly-inspo.md

```diff
@@ -0,0 +1,301 @@
+
+## Blurb
+
+`xplr` will act as a central hub for all of our inspiration. Every week, members of 
+the dev team will make a pull request to the `xplr` repository containing something 
+that inspired them. Their inspiration can be anything from a cool CSS property, an 
+interesting design, quirky libraries, or exciting frameworks. During our dev meeting, 
+we will talk about the stuff we found and at the end of the discussion, update a 
+`CHANGE_LOG` to keep a weekly timeline of the inspo we added.
+
+The pull request will be in the form of a markdown file that contains a description of 
+the inspo, why they think it's interesting, and how we could potentially use it in the 
+future. The file must also include metadata to help tag and catalog who the inspo came 
+from and what category the inspo belongs to.
+
+All these markdown files will accumulate into a blog site that is easily accessible to
+anyone outside of the dev team to read. The site will be searchable, filterable, and 
+creatively designed to show off our inspo in an unique fashion.
+
+Ultimately, the purpose of `xplr` is to constantly expose our dev team to the greater 
+world of dev and to familarize everyone with documentation, the PR process, and 
+exploration.
+
+## Why are we doing this?
+
+One of the things that we really stress at Creative Labs is having taste and 
+inspiration. Inspo is a big part of creativity and allows us to create better work. 
+Although inspo is usually something that is associated with the design side, exposing 
+ourselves to different dev concepts would make us better developers and empower us to 
+challenge our designers to make even more creative work.
+
+## What is the high level design/implementation?
+
+`xplr` can be built in an entirely self-contained environment. `xplr` will be built
+as a `Next.js` application that will be deployed to Netlify. There isn't a need for
+any servers as `Next.js` comes with an `api` that Netlify will deploy as a AWS function
+for us.
+
+### Organization
+
+The `xplr` repository will have a standard `Next.js` structure. On top of the standard
+configuration, we will also need to have a `_posts` and `_team` directory to keep 
+track of each inspo post and each team member. See [Appendix](#organization-(expanded))
+for more details on organization.
+
+### Verification
+
+In order to reduce the chance of introducing bugs into our application. We will need
+to write some custom linters to test the efficacy of the our posts.
+
+**Posts Linter**
+
+* Check that the author exists in the team
+* `title` is identical to the title of the article
+* `slug` is identical to file name
+* Check that `tag` is delimited white spaces and is an available tag in the'
+  `_posts/tags.yml` file
+
+**Team Members Linter**
+
+* Check if `name` is capitalized
+* Check that the file name is the `name` but in `kebab-case`
+
+**Markdown Linter**
+
+* Make sure all markdown files are only 80 characters wide
```

Comment:
minor note but i personally think 100 characters as a soft limit (ie throw up a warning) and 120 characters as a hard limit (ie throw up an error) is better. 80 characters is pretty short especially given screen sizes today

Reply:
Hmmm somehow I got the number 80 in my head.. I think I read somewhere that 80 character width is the gold standard..

https://stackoverflow.com/questions/110928/is-there-a-valid-reason-for-enforcing-a-maximum-width-of-80-characters-in-a-code

But in terms of UX, I heard that generally you should go from 60 - 75!

Reply:
yeah i also have heard the same, but i remember reading some blog post in support of 100-120 char widths somewhere and it convinced me LOL but it's not important

also re UX, i think that's more for display text, like when text shows up on a website or something. i thought this bit was for the text width of the actual file when viewed in a text editor like vim (ie in a relatively small monospaced font). but yeah it's not important, just a personal qualm

Reply:
okie yeah i think i prefer 80? mainly bc with vscode i think that's where most screen sizes will cut it off!